Automattic/wp-calypso
# Login: Move the login component out of the auth section

This pull request fixes #12668 by moving the new login work to a different section, making it easier to keep exceptions in place for the auth section (for example no masterbar).


#### Testing instructions
  
1. Run `git checkout move/login` and start your server
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Check that it loads with a masterbar at the top
4. Update `config/development.json`, setting `features.oauth` to true
5. Open the [`Oauth Login` page](http://calypso.localhost:3000/oauth-login)
6. Check that the desktop app login loads correct.
  
#### Additional notes
  
This will probably require changes to the desktop app, but I haven't tested it yet.
@jblz This might also break magic links.
  
#### Reviews
  

   
@Automattic/sdev-feed
